languages/java/app-encryption/README.md

```diff
@@ -108,6 +109,23 @@ CryptoPolicy basicExpiringCryptoPolicy = BasicExpiringCryptoPolicy.newBuilder()
     .build();
 ```
 
+#### (Optional) Enable Session Caching
+
+Session caching is disabled by default. If there is a use case where multiple users/threads are trying to access 
+the same session after some operations, you can enable session cache.
```

Comment:
this reads oddly. suggest clarifying/rewording the use case for session caching, similar to above comment